app-frontend/src/app/components/pages/project/analyses/analyses/index.js

```diff
@@ -218,7 +219,7 @@ class AnalysesListController {
             .then(this.autoGenerateRenderDef.bind(this))
             .then(() => {
                 const tileUrl = this.analysisService.getAnalysisTileUrl(analysisId);
-                return L.tileLayer(tileUrl, { maxZoom: 30 });
```

Comment:
Turns out that the maxZoom is required - it defaults to 18 otherwise. By setting maxZoom to 30 and maxNativeZoom to a lower number, we let them zoom in futher without fetching more tiles.

Reply:
@Lknechtli this is a good suggestion. I will update it.